extensions/wikia/Email/EmailController.class.php

```diff
@@ -34,6 +38,7 @@ public function init() {
 			$this->currentUser = $this->findUserFromRequest( 'currentUser', $this->wg->User );
 			$this->targetUser = $this->findUserFromRequest( 'targetUser', $this->wg->User );
 			$this->test = $this->getRequest()->getVal( 'test', false );
+			$this->marketingFooter = $this->request->getVal( 'marketingFooter' );
```

Comment:
We could, but mustache doesn't care if a value is false, null, 0, undefined, etc. If it's falsey it won't render the conditional content. I'm inclined to leave it as is but if you'd prefer to make typing more explicit I suppose we could update this.